scripts/sampinctrl.py

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) 2021 Teslabs Engineering S.L.
+# Copyright (c) 2022 Gerson Fernando Budke
+# SPDX-License-Identifier: Apache 2.0
+
+"""
+Utility to autogenerate pinctrl definitions.
+
+Usage::
+    python3 sampinctrl.py [-i /path/to/configs] [-o /path/to/include]
+
+Copyright (c) 2021 Teslabs Engineering S.L.
+Copyright (c) 2022 Gerson Fernando Budke
+SPDX-License-Identifier: Apache 2.0
+"""
+
+import argparse
+from collections import OrderedDict
+from pathlib import Path
+from natsort import natsorted
+import re
+
+import yaml
```

Comment:
natsort can be moved together with yaml (it's not from std Python lib)

Reply:
Done!